docs/userguide/objective.md

```diff
@@ -1,24 +1,77 @@
-* **A list of targets:** The list of targets that are optimized. Note that the ``SINGLE`` mode also requires a list containing the single target. See [here](./targets) for details.
+To create an objective, it is necessary to provide the following:
+* An optimization `mode`: This can be either `SINGLE` or `DESIRABILITY`,
+  denoting the optimization of a single target function or a combination of
+  different target functions respectively.
+* A list of `targets`: The list of targets that are optimized. Note that the
+  `SINGLE` mode also requires a list containing the single target.
 
-## Supported Optimization Modes
+```{note}
+We are actively working on adding more objective modes for multiple targets.
+```
 
+## Supported Optimization Modes
 Currently, BayBE offers two optimization modes.
 
-### Single Target Optimization: The ``SINGLE`` mode
+### SINGLE
+In the `SINGLE` mode, objectives focus on optimizing a single target. 
+Nearly all of the [examples](../../examples/examples) use this objective mode.
+
+```python
+from baybe.targets import NumericalTarget
+from baybe.objective import Objective
+
+target1 = NumericalTarget(name="yield", mode="MIN", bounds=(0, 100))
+objective = Objective(mode="SINGLE", targets=[target1])
+```
+
+### DESIRABILITY
+The ``DESIRABILITY`` mode enables you to combine multiple targets via scalarization 
+into a single value, a method well know and also utilized in classical DOE.
+
+Besides `mode` and `targets`, this objective type has two additional optional arguments:
+* `weights`: Some targets might be more important than others.
+  You can specify the relative weights of the targets in this argument.
+  BayBE automatically normalizes the numbers you provide, so only the relations between 
+  them matter.
+* `combine_func`: Specifies the function used for combining the transformed targets. 
+  You can choose between `MEAN` and `GEOM_MEAN`, referring to the arithmetic and 
+  geometric mean respectively.
+
+The definitions of the means are as follows, where $t_i$ enumerate the **scaled** target 
+observations for a single measurement and $w_i$ are the weights associated with the 
+respective target:
+
+$$
+\text{MEAN} &= \frac{1}{\sum w_i}\sum_{i} w_i \cdot t_i \\
+\text{GEOM_MEAN} &= \left( \prod_i t_i^{w_i} \right)^{1/\sum w_i}
+$$
 
-In the ``SINGLE`` mode, objectives focus on maximizing or minimizing a single target. 
-Nearly all of the examples use this objective mode.
+```{attention}
+Due to the combination of targets of potentially different scale, in `DESIRABILITY` 
+objective mode, all provided targets must have `bounds` specified so they can be 
+normalized before being combined.
+```
 
-### Combining multiple targets: The ``DESIRABILITY`` mode
+In the example below we use three different targets (which all have a different goal) 
```

Comment:
Again: Passive vs. active, this time with the "we".

Reply:
what is passive here?

Or do you mean the user-addressing via `we` ?

Reply:
Yes, that is what I mean. Probably also just an issue of me preferring a different style of writing or a change of style within the file that I observed while reading. So no urgent need to fix if you prefer your variant.